gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
These two `gradle/` files are still showing up.

Reply:
Made the requested change here.